Makefile

```diff
@@ -24,3 +24,10 @@ HEROKU_APP_US = $(SERVICE_SYSTEM_CODE)-us
 GRAFANA_DASHBOARD = $(SERVICE_SYSTEM_CODE)
 
 export GITHUB_RELEASE_REPO := Financial-Times/$(SERVICE_SYSTEM_CODE)
+
+local-build:
+	obt build --build-folder="./public/" --sass="./src/main.scss" --js="./src/main.js"
```

Comment:
Having an extra Make command which watches also for doing dev work would be great!

Reply:
I'm not sure if there's a way to pass a watch flag through to obt or whether a command like this is needed instead:
```
watch-build:
    obt build --build-folder="./public/" --sass="./src/main.scss" --js="./src/main.js" --watch
```

Reply:
Agree we should have a watch one :) reckon as a separate PR though to keep this one focused

Reply:
I've just noticed, not that this is exactly a problem, but in order for this to watch and for the server to run, we've got to have two different processes running. Are we cool with that?